### ISSUE TITLE
Fix specs (pending lutaml-model support of CDATA)

### ISSUE THREAD

**Issue by ronaldtse** [relaton/ieee-idams#2]
The specs are currently failing on missing CDATA content, which will be done here:
* https://github.com/lutaml/lutaml-model/issues/116

```
  1) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Guide for Information Technology Software Reuse-Concept of Operations for Interoperating Reuse Libraries [Developed in cooperation with the Reuse Library Interoperability Group (RIG)]</standardsfamilytitle>
               actual node  : <title><![CDATA[IEEE Std 1430-1996]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  2) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Standard Requirements for Direct Current Instrument Shunts</standardsfamilytitle>
               actual node  : <title><![CDATA[IEEE Std 316-1971]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  3) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Standard Requirements, Terminology, and Test Procedures for Neutral Grounding Devices</standardsfamilytitle>
               actual node  : <title><![CDATA[ANSI/IEEE Std 32-1972]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  4) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>American National Standard Graphic Symbols for Electrical Wiring and Layout Diagrams Used in Architecture and Building Construction</standardsfamilytitle>
               actual node  : <title><![CDATA[ANSI Std Y32.9-1972]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  5) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Standard Specification Format Guide and Test Procedure for Linear, Single-Axis, Pendulous, Analog Torque Balance Accelerometer</standardsfamilytitle>
               actual node  : <title><![CDATA[IEEE Std 337-1972]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  6) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Recommended Practice for Personnel Qualifications for Installation and Maintenance of Stationary Batteries</standardsfamilytitle>
               actual node  : <title><![CDATA[IEEE Unapproved Draft Std P1657_D26, Mar 2009]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  7) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Reduced-Pin and Enhanced-Functionality Test Access Port and Boundary-Scan Architecture</standardsfamilytitle>
               actual node  : <title><![CDATA[IEEE Std 1149.7-2009]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  8) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>American National Standard Recommended Practice for an On-site, Ad-Hoc Test Method for Estimating Radiated Electromagnetic Immunity of Medical Devices to Specific Radio Frequency Transmitters</standardsfamilytitle>
               actual node  : <title><![CDATA[ANSI C63.18-2014 (Revision of ANSI C63.18-1997)]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  9) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
     Failure/Error: expect(output).to be_analogous_with(input)
     
       DIFF 1: expected node: <standardsfamilytitle>ISO/IEC/IEEE International Standard for Systems and Software Engineering -- Life Cycle Management -- Part 4: Systems Engineering Planning</standardsfamilytitle>
               actual node  : <title><![CDATA[ISO/IEC/IEEE 24748-4 First edition 2016-05-15]]></title>
               diff from    : standardsfamilytitle
               diff to      : title
     # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  10) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Safety Levels with Respect to Human Exposure to Electric, Magnetic, and Electromagnetic Fields, 0 Hz to 300 GHz</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE Std C95.1-2019/Cor2-2020 (Corrigenda to IEEE Std C95.1-2019)]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  11) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>ISO/IEC/IEEE International Standard - Systems and software engineering--Life cycle management--Part 3: Guidelines for the application of ISO/IEC/IEEE 12207 (software life cycle processes)</standardsfamilytitle>
                actual node  : <title><![CDATA[ISO/IEC/IEEE 24748-3:2020(E)]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  12) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>ISO/IEC/IEEE International Standard - Systems and software engineering--Software life cycle processes--Part 2: Relation and mapping between ISO/IEC/IEEE 12207:2017 and ISO/IEC 12207:2008</standardsfamilytitle>
                actual node  : <title><![CDATA[ISO/IEC/IEEE 12207-2:2020(E)]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  13) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Information technology--Telecommunications and information exchange between systems Local and metropolitan area networks--Specific requirements - Part 11: Wireless LAN Medium Access Control (MAC) and Physical Layer (PHY) Specifications</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE Std 802.11-2020 (Revision of IEEE Std 802.11-2016)]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  14) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Intercloud Interoperability and Federation (SIIF)</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE Std 2302-2021]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  15) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Approved Draft Guide for Establishing Short Circuit Withstand Capabilities of Liquid Immersed Power Transformers, Regulators, and Reactors</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE Std C57.164-2021]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  16) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Third-Generation Video Coding</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE Std 1857.10-2021]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

  17) Ieee::Idams::Publication round-trips IEEE IDAMS record: ./spec/ieee/idams/publication_spec.rb[1]
      Failure/Error: expect(output).to be_analogous_with(input)
      
        DIFF 1: expected node: <standardsfamilytitle>IEEE Standard for Logistics Operation Process in a Smart Factory</standardsfamilytitle>
                actual node  : <title><![CDATA[IEEE P2934/D1, May 2021]]></title>
                diff from    : standardsfamilytitle
                diff to      : title
      # ./spec/ieee/idams/publication_spec.rb:19:in `block (3 levels) in <top (required)>'

Finished in 0.13454 seconds (files took 0.11958 seconds to load)
18 examples, 17 failures
```

**Comment by ronaldtse** [comment]
Once https://github.com/lutaml/lutaml-model/issues/116 is done then these specs should be automatically fixed.

**Comment by ronaldtse** [comment]
Fixed in #4 .

**Comment by ronaldtse** [comment]
We now parse all ieee-rawbib data! FYI @andrew2net
